### PR TITLE
integration with Travis-CI tool for better pull request checking before merging

### PR DESCRIPTION
**Pull request by olisikh**

### CODE REVIEW

**Comment by olisikh** [comment]
Oh god, this is harder than I thought, probably much easir would be to create a Docker image for tests and run it on Travis, instead of directly configure Travis to install all needed services (basically only one - mysql) and run the tests, it fails quite unexpectedly...

Having a Docker image would simplify configuration, as we'd not need travis-ci profile for spring-boot any longer

**Review by artemlisikh — DISMISSED** [review]
nice job!

**Comment by olisikh** [comment]
For now everything works, but the command to run tests require an adjusted command:
mvnw clean test -P test

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: java
+jdk:
+  - oraclejdk8
+services:
+  - mysql
+sudo: true
+cache:
+  directories:
+  - $HOME/.m2
+
```
Comment (konteno) [inline .travis.yml]: what's cache for?

**Review by artemlisikh — APPROVED** [review]

**Pull request merged**